assets/src/edit-story/elements/media/edit.js

```diff
@@ -132,15 +138,27 @@ function MediaEdit({ element, box }) {
     ref: setCroppedMedia,
     draggable: false,
     src: resource.src,
-    alt: resource.alt,
+    alt: __('Drag to move media element', 'web-stories'),
     opacity: opacity / 100,
+    tabIndex: 0,
```

Comment:
I really don't know how we can make it clear, that the image has focus, unfortunately. It already has a border. We could apply some sort of filter to it? Not sure what to do and I don't think it's important for this PR for now.

Reply:
Oh, and regarding the fact that in Safari you cannot tab from the image to the slider, is due to Safari's default of not actually using tab to navigate elements. You have to enable that in settings (which I assume all keyboard-using Safari users do):

<img width="507" alt="Screen Shot 2020-08-20 at 11 56 03" src="https://user-images.githubusercontent.com/637548/90796311-22570700-e2dd-11ea-8a90-8cd99dc65cad.png">

Reply:
> Not sure what to do and I don't think it's important for this PR for now.

Probably just something to flag Sam with at some point, let's ignore for now.